distribution/pom.xml

```diff
@@ -73,6 +73,44 @@
       <version>0.5.1-SNAPSHOT</version>
       <type>pom</type>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>5.9.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I've introduced a migration script of exising data - and saw there were no tests in this module.
So I've added a test, along with required machinery (with `<scope>test</scope>`)  to run it